### PR TITLE
Update minor dependencies and align npm scripts

### PR DESCRIPTION
**Pull request by jobvs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "description": "Displays a progress in a circle",
   "scripts": {
     "start": "grunt",
-    "checkTypes": "tsc --noEmit",
+    "typeCheck": "tsc --noEmit",
```
Comment (Andries-Smit) [inline package.json]: Pre commit, should also be renamed

Comment (jobvs) [reply package.json]: Fixed. Strange because I was able to commit...

Comment (Andries-Smit) [reply package.json]: missing npm dependency `pre-commit`

Comment (jobvs) [reply package.json]: Now it all makes sense

**Pull request merged**